source/shared/cpp/ObjectModel/ParseContext.h

```diff
@@ -21,16 +21,24 @@ namespace AdaptiveSharedNamespace
         std::shared_ptr<ElementParserRegistration> elementParserRegistration;
         std::shared_ptr<ActionParserRegistration> actionParserRegistration;
         std::vector<std::shared_ptr<AdaptiveCardParseWarning>> warnings;
-
         // Push/PopElement are used during parsing to track the tree structure of a card.
         void PushElement(const std::string& idJsonProperty,
                          const AdaptiveSharedNamespace::InternalId& internalId,
                          const bool isFallback = false);
         void PopElement();
 
+        void SetLanguage(const std::string& value);
+        std::string GetLanguage() const;
+
+        ContainerStyle GetParentalContainerStyle() const;
+        AdaptiveSharedNamespace::InternalId GetIDOfParentWithPadding(void)const;
```

Comment:
>GetIDOfParentWithPadding [](start = 44, length = 24)

This should maybe be `PaddingParentInternalId()`

Reply:
yes, ParentInternalId soulds better. But PaddingParent? sounds weird to me. :-) This method returns the id of a closest parent that has padding. 

---
In reply to: [259080203](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259080203) [](ancestors = 259080203)

Reply:
i will mark it resolved for now. and feel free to change it.

---
In reply to: [259200833](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259200833) [](ancestors = 259200833,259080203)